go/filemanager/pfsmod/chunkmeta.go

```diff
@@ -0,0 +1,221 @@
+package pfsmod
```

Comment:
`chunkmeta` definations should also add to design doc: https://github.com/PaddlePaddle/Paddle/tree/develop/doc/design/file_manager